src/plugins/intel_cpu/src/emitters/plugin/x64/jit_bf16_emitters.hpp

```diff
@@ -114,14 +178,24 @@ class jit_uni_vcvtneps2bf16 : public jit_emitter {
         push_arg_entry_of("rounding", 0x00010000, true);
         push_arg_entry_of("selector", selector_int32, true);
         push_arg_entry_of("mask_truncation_word", 0x0000ffff, true);
+        push_arg_entry_of("bf16_max", 0x7F7F0000, true);
+        push_arg_entry_of("bf16_min", 0xFF7F0000, true);
     }
 
     size_t aux_vecs_count() const override {
-        if (dnnl::impl::cpu::x64::mayiuse(dnnl::impl::cpu::x64::avx512_core_bf16))
-            return 0;
-        return host_isa_ == dnnl::impl::cpu::x64::avx512_core ? 2 : 1;
+        if (mode_ == arithmetic_mode::saturation) {
+            if (host_isa_ == dnnl::impl::cpu::x64::cpu_isa_t::avx512_core) {
+                return 5;
+            } else if (dnnl::impl::cpu::x64::mayiuse(dnnl::impl::cpu::x64::cpu_isa_t::avx2_vnni_2) ||
+                       dnnl::impl::cpu::x64::mayiuse(dnnl::impl::cpu::x64::cpu_isa_t::avx512_core_bf16)) {
+                return 3;
+            }
+            return 4;
```

Comment:
Should update(minimize) the number after modify implementation, for perf and snippets scalability.

Reply:
Done